docs/plugins/02-available-plugins.md

```diff
@@ -362,20 +362,11 @@ Every plugin that works with Neovim works with LunarVim, here are some examples
 ``` lua
 {
   "ray-x/lsp_signature.nvim",
-  event = {"InsertEnter"},
+  event = "InsertEnter",
   config = function()
     require "lsp_signature".setup()
```

Comment:
this should be `require "lsp_signature".on_attach()`
can you fix that as well :pleading_face:

Reply:
![image](https://user-images.githubusercontent.com/10992695/128470741-ad240f14-af91-4959-a2a1-133007d5261b.png)

Reply:
What is the difference of `.on_attach()` vs on `.setup()` (cause `.setup()` is working right for me now, and the documentation on the lsp_signature website seems to use `.setup`)

Reply:
you are right, apparently setup is the better option